js/expiring_messages.js

```diff
@@ -11,8 +11,8 @@
         var expired = new Whisper.MessageCollection();
         expired.on('add', function(message) {
             console.log('message', message.get('sent_at'), 'expired');
-            message.destroy();
             message.getConversation().trigger('expired', message);
+            message.destroy();
```

Comment:
It's not obvious to me why this ordering is significant. If it is can you leave a comment?